package.json

```diff
@@ -83,9 +83,9 @@
     "semantic-release": "^17.4.4",
     "storybook-readme": "^5.0.8",
     "tap-xunit": "^2.4.1",
-    "ts-jest": "^25.4.0",
-    "ts-loader": "^7.0.1",
-    "typescript": "^3.8.3"
+    "ts-jest": "^27.1.4",
+    "ts-loader": "^8.1.0",
```

Comment:
Can't go to `ts-loader` 9 yet as it has a minimum webpack version of `5` (we are on `4`) https://github.com/TypeStrong/ts-loader/releases/tag/v9.0.0.  Moving to the newer version of webpack will be part of the storybook PR.